src/platform/windowing/windowing.h

```diff
@@ -40,9 +40,11 @@
 
 #if defined(_WIN32) || defined(_WIN64)
 #include "win32.h"
-#elif defined(__android__)
+#elif defined(__ANDROID__)
```

Comment:
I know this is still a draft, but thought I'd drop this in case it was missed:

This is likely defined by the NDK, we could also make this:
```c
#elif defined(__android__) || defined(__ANDROID__)
```